sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/tests/Azure.Monitor.OpenTelemetry.LiveMetrics.Tests/DocumentTests/HttpClientDependecyTests.cs

```diff
@@ -0,0 +1,118 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Azure.Monitor.OpenTelemetry.LiveMetrics.Internals;
+using Azure.Monitor.OpenTelemetry.LiveMetrics.Models;
+using Microsoft.AspNetCore.Builder;
+using OpenTelemetry;
+using OpenTelemetry.Trace;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Azure.Monitor.OpenTelemetry.LiveMetrics.Tests.DocumentTests
+{
+    public class HttpClientDependecyTests : DocumentTestBase
+    {
+        private const string TestServerUrl = "http://localhost:9996/";
+
+        public HttpClientDependecyTests(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        [Fact]
+        public void VerifyHttpClientAttributes()
+        {
+            // SETUP
+            var uniqueTestId = Guid.NewGuid();
+
+            var activitySourceName = $"activitySourceName{uniqueTestId}";
+            using var activitySource = new ActivitySource(activitySourceName);
+            var listener = new ActivityListener
```

Comment:
We should rely on OpenTelemetry provider instead of creating a listener. This could be a follow up task.